crt_portal/cts_forms/migrations/0074_auto_20200602_1417.py

```diff
@@ -0,0 +1,101 @@
+# Generated by Django 2.2.12 on 2020-06-02 18:17
```

Comment:
One small thing is we like to replace the auto generated migration names with something descriptive,
so, this could be  0074_add_response_templates or something like that so the migration files are scannable by name.

Reply:
D'oh, yes, I forgot to do this. Adding and bumping up since trends is now `074`